requirements.txt

```diff
@@ -1,4 +1,25 @@
+atomicwrites==1.4.0
```

Comment:
Not sure why there are so many new modules. Most of these seem like dependencies. Can you please clean up to have only the modules needed?

Reply:
Removed all unnecessary modules.